packages/llopt/llopt.1.0.0/opam

```diff
@@ -10,7 +10,7 @@ build: ["jbuilder" "build" "-p" "llopt" "-j" jobs]
 depends: [
   "ocaml"
   "jbuilder" {>= "1.0+beta7"}
```

Comment:
```suggestion
  "jbuilder" {>= "1.0+beta17"}
```
Not sure why the lower bound fails or what is the correct one. This suggestion is for a version of jbuilder that was released a couple of months before this project was released

Reply:
we can fix that later in another PR